app/src/org/commcare/models/AndroidSessionWrapper.java

```diff
@@ -242,11 +242,9 @@ public EvaluationContext getEvaluationContextWithAccumulatedInstances(String com
     }
 
     @Override
-    public void prepareExternalSources(RemoteInstanceFetcher fetcher)
-            throws RemoteInstanceFetcher.RemoteInstanceException {
-        for(StackFrameStep step : session.getFrame().getSteps()) {
-            step.initDataInstanceSources(fetcher);
-        }
+    public void prepareExternalSources() {
```

Comment:
This is unused at this point on Android and since there is no implementation for `RemoteInstanceFetcher` currently in CC Android, I am now just throwing an exception if we call this on Android.